members/README.md

```diff
@@ -59,3 +60,34 @@ Returns all members in the system.
 - **Error Response:**
   - **Code:** 500
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'Something went wrong. Please contact admin' }`
+
+## **GET /members/idle**
+
+Returns all inactive/idle members in the system.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  None
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+
+```
+{
+  message: 'Idle members returned successfully!'
+  members: [
+           <id>
+         ]
```

Comment:
This is supposed to an array for `username`(s)
Because on frontend we would display these members

The `id` field in whole of backend refers to firestore generated id which is long string
`username` refers to RDS username of the members